wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -11156,7 +11167,7 @@
 					"@executable_path/Frameworks",
 					"@executable_path/../../Frameworks",
 				);
-				MARKETING_VERSION = 1.0;
+				MARKETING_VERSION = 3.112.1;
```

Comment:
WIRE_SHORT_VERSION

Reply:
I don't understand yet, why there is a warning only for the `Wire Notification Service Extension` so I reverted the changes.